lib/internal/process/esm_loader.js

```diff
@@ -23,13 +22,6 @@ exports.initializeImportMetaObject = function(wrap, meta) {
 };
 
 exports.importModuleDynamicallyCallback = async function(wrap, specifier) {
-  assert(calledInitialize === true || !userLoader);
```

Comment:
This assertion seems unrelated, should it be retained? Not sure how likely that mistake is.

Reply:
The `calledInitialize` variable was entirely removed because this was what tracks if the warning has already been shown.